plugin.css

```diff
@@ -1,65 +0,0 @@
-.pluginContainer {
-        font-family: Arial;
-        padding: 10px;
-    }
-    
-    h1 {
```

Comment:
styling by tag might interfere with global styles. It would be best to use classess with some prefix specific for this plugin
like: laurence-the-great__title 😃 etc.